User_Stories.md

```diff
@@ -0,0 +1,11 @@
+﻿## User Stories
+
+As an end user, I want to be able to upload shapes or symbols for the artificial intelligence to detect so that I may create new targets on the fly.
+
+As an end user, I want to be able to select a symbol from a list of symbols so that the artificial intelligence may target the symbol with the laser.
+
+As an end user, I want the artificial intelligence to detect only ***balloons*** that have symbols on them so that random objects do not get burnt.
+
+As an end user, I want to be able to activate and deactivate the laser so that I can avoid burning holes in things that should not have a hole in them.
```

Comment:
Change to something more focused around having a kill switch/cancel button or the likes. User shouldn't be able to turn on the laser but should have control over stopping the program at any point.